e2e-tests/tests/headlampPage.ts

```diff
@@ -104,7 +104,9 @@ export class HeadlampPage {
     const rowsDisplayed1 = await this.getRowsDisplayed();
 
     // Click on the next page button
-    const nextPageButton = this.page.getByTitle('Next page');
+    const nextPageButton = this.page.getByRole('button', {
+      name: 'Go to next page',
```

Comment:
Why was this text changed?

Reply:
new table uses a bit different wording in its' pagination